README.md

```diff
@@ -16,8 +16,50 @@ This guide aims to improve the way your team uses [jQuery](http://jquery.com/).
 ## Table of Contents
 
 * [About jQuery](#about-jquery)
-
+* [Avoid using `.animate()` method when possible](#avoid-using-`.animate()`-method-when-possible)
```

Comment:
Incorrect slug. Should be [`#avoid-using-animate-method-when-possible`](https://github.com/voorhoede/jquery-style-guide/blob/avoid-animate-method/README.md#avoid-using-animate-method-when-possible).